Dockerfile

```diff
@@ -1,20 +1,27 @@
-FROM nginx:1.14-alpine
+FROM nginx:1.16-alpine
 
-RUN apk --no-cache add jq
+RUN apk --no-cache add jq findutils gzip
 
-ADD dist /www
-ADD package.json /
+COPY dist /www
 
 RUN rm -r /etc/nginx/conf.d
 RUN mkdir -p /etc/nginx/config
-ADD nginx.conf /etc/nginx/config
+COPY nginx.conf /etc/nginx/config
 
-ADD VERSION /
-ADD scripts/start.sh /
+COPY VERSION /
+COPY scripts/start.sh /
 
 # Test certifiates will be overwritten in production by configmap
-ADD test/certs /etc/nginx/certs
+COPY test/certs /etc/nginx/certs
```

Comment:
What about this? Was this intended to be used with https? Could we remove this also?

Reply:
This is a self signed cert for when you run this container on your own machine, that way you can at least get a response from it. Otherwise nginx doesn't start up since it complains about the missing files.

Also I think we need it for the 'smoke test' in the CI to work. (If that is still there). I think there is a step where we just check if the container starts up and is able to serve a request.

Reply:
In the current configuration, the nginx container serves plain http. The CI configuration does not contain https checking and the deployment in the helm chart does not inject anything into the pod. I've checked the pods of happa on gauss, the certificate & key there is the same that is copied here into the container.

For me both is now fine: Removing this or fixing https serving from the happa container. What do you suggest?

Reply:
For the record: [This commit] removes the ssl stuff from nginx config

[This commit]: https://github.com/giantswarm/happa/commit/890aeb3953657c822f53bdb6be680d7073daf7d2#diff-daaf2fceca49ed034eace91c4e875653https://github.com/giantswarm/happa/commit/890aeb3953657c822f53bdb6be680d7073daf7d2#diff-daaf2fceca49ed034eace91c4e875653

Reply:
Oh! 😅 yeah it looks like these files are totally not needed then!

Reply:
:) alright I'll remove this then